nomad/client_meta_endpoint.go

```diff
@@ -0,0 +1,69 @@
+package nomad
+
+import (
+	"time"
+
+	metrics "github.com/armon/go-metrics"
+	log "github.com/hashicorp/go-hclog"
```

Comment:
```suggestion
	"github.com/armon/go-metrics"
	"github.com/hashicorp/go-hclog"
```

Reply:
Huh... wonder if my editor is misconfigured. Thanks.